src/common/Ingame/Dashboard.tsx

```diff
@@ -0,0 +1,40 @@
+import needle from '@/asssets/ingame/needle.png';
+
+interface DashboardProps {
+  type: 'wpm' | 'accuracy';
+  value: number;
+}
+const Dashboard = ({ type, value }: DashboardProps) => {
+  const ratioStyle = new Map([
+    [
+      'wpm',
+      `${value > 800 ? '225deg' : `${Math.floor((value * 180) / 800) + 45}deg`}`,
+    ],
+    ['accuracy', `${value > 100 ? '225deg' : `${value * 1.8 + 45}deg`}`],
+  ]);
+
+  return (
+    <>
+      <div>
+        <div className='w-[24rem] h-[12rem] rounded-[12rem_12rem_0_0] border-x-[2rem] border-t-[2rem] border-green-100 box-border'>
+          <img
+            src={needle}
+            className={`w-[6rem] relative top-[8rem] left-[5rem] origin-[80%_15%]`}
+            style={{
+              rotate: ratioStyle.get(type),
+            }}
+          />
```

Comment:
alt 속성도 추가하시면 더욱 좋을 것 같습니다! 😊
```suggestion
          <img
            src={needle}
            alt='계기판'
            className={`w-[6rem] relative top-[8rem] left-[5rem] origin-[80%_15%]`}
            style={{
              rotate: ratioStyle.get(type),
            }}
          />
```